omod/src/main/webapp/pages/metadata/encounters/encountertypes/templates/edit.gsp

```diff
@@ -18,11 +18,20 @@
         ${ui.message('general.onDate')}
         {{ encounterType.auditInfo.dateChanged | serverDate }}
     </p>
+    
+    <p>
+        ${ui.message('adminui.location.address')}:
+    </p>
 </fieldset>
 
 
 <form class="simple-form-ui" name="encounterTypeForm" novalidate ng-submit="save()">
     <p>
+        <label>${ui.message('adminui.location.address')}</label>
+        <input ng-model="encounterType.location" required/>
+    </p>
```

Comment:
Why are you adding this inout, I thought the ticket says adding view/edit privilege fields